src/main/webapp/app/overview/page-discussion-section/page-discussion-section.scss

```diff
@@ -64,6 +64,11 @@ $label-color: #6a737d;
     }
 }
 
+.post-result-information {
```

Comment:
This is duplicated from course-discussion.scss, perhaps it would make sense to have these styles in a more global, metis-wide file

Reply:
I added such as file, it's a good idea, maybe I find more styles, that can be added to the `metis.component.scss` file when creating the next PRs.